### ISSUE TITLE
Comparison to wastebin

### ISSUE THREAD

**Issue by yonas** [WantGuns/bin#28]
How does this compare to https://github.com/matze/wastebin ?

**Comment by Tofee** [comment]
It looks like wastebin uses an sqlite database as a storage backend. For me it's a no-go...

**Comment by matze** [comment]
Also as a bit of historical background: I took the design, general flow and ultra lightweight principle from bin (and also acknowledge that in the README.md). But since then it has developed further and yes, it uses a SQLite database to store pastes and metadata instead of dumping everything as flat files to disk.

**Comment by Tofee** [comment]
@matze I realize I might have sounded a bit harsh in my comment; let me rephrase a bit. Wastebin looks like a very good variation of bin, and the latter seems more of less EOL (I even added my own patches in my copy of the repo). It's just that I want my pastes to be as simple as possible, hence my no-go to a database.

One regret I have from bin (and I don't think it's available in wastebin either?) is that there is no way to customize the paste name. As a result it is difficult to remember what a previous paste was about.

**Comment by matze** [comment]
> I realize I might have sounded a bit harsh in my comment

No offense taken, all good :+1:

Also: I initially intended to stay with a filesystem approach but a few things like encryption, burn-after-reading/expiration etc. require something else on top.

> One regret I have from bin (and I don't think it's available in wastebin either?)

The name (but not the URL) can be set since 2.6.0.
